vm/src/tests/cairo_run_test.rs

```diff
@@ -1032,3 +1032,26 @@ fn divmod_igcdex_not_one() {
     let error_msg = "Operation failed: divmod(1, 340282366920938463463374607431768211457, 340282366920938463463374607431768211457), igcdex(340282366920938463463374607431768211457, 340282366920938463463374607431768211457) != 1";
     run_program_with_error(program_data.as_slice(), error_msg);
 }
+
+fn cairo_run_print_felt() {
+    let program_data = include_bytes!("../../../cairo_programs/print_felt.json");
+    run_program_simple(program_data);
+}
+
+#[test]
+fn cairo_run_print_array() {
+    let program_data = include_bytes!("../../../cairo_programs/print_array.json");
+    run_program_simple(program_data);
+}
+
+#[test]
+fn cairo_run_print_dict_felt() {
+    let program_data = include_bytes!("../../../cairo_programs/print_dict_felt.json");
+    run_program_simple_with_memory_holes(program_data, 5);
+}
+
+#[test]
+fn cairo_run_print_dict_array() {
+    let program_data = include_bytes!("../../../cairo_programs/print_dict_array.json");
+    run_program_simple_with_memory_holes(program_data, 4);
+}
```

Comment:
These tests need to be labeled to build only when the `print` feature is enabled. Otherwise they'll fail due to missing hints when it isn't.

Reply:
Also, I'm confused about how to test that we actually see any input in `stdout`, which is the desired effect.

Reply:
I'm not sure either, added those to be sure that the hint wasnt failint, I was visually checking that printing was ok. I could check if it's possible to get the stdout to compare

Reply:
It would be good. Let's also see what the others think, maybe I'm being too pedantic here and we can do with visual checking for this.
A quick search leads me to think this is not easy to solve via unit tests, but at some point we may need to start doing some snapshot testing and diffing the actual output or something.
@juanbono @pefontana @fmoletta

Reply:
I think it is a good idea to check the printed value, but I really don't know any easy way to do it. If there is a fast way to check it would be great, in case it is too complex we can merge the PR as it is

Reply:
I think there might be one solution: update the signature of the hint in order to take a writer. In tests I can pass a buffer and during run I can pass io::stdout. Would that be a good solution or would you just prefer to keep as is for now?

Reply:
I think it is better to avoid the writer, since the purpose of the hint dont need it. We can check the print manually